pages/index.tsx

```diff
@@ -3,9 +3,12 @@ import Layout from "../components/Layout";
 import Socials from "../components/Socials";
 import logo from "../public/code-youngstown-white-background-final.png";
 
+const pageDescription =
+  "Code Youngstown, connecting software engineers, developers, and coders in the Youngstown, OH area.";
```

Comment:
This can probably just be

`Connecting software engineers, developers, and coders in the Youngstown, OH area.`